frontend/package.json

```diff
@@ -0,0 +1,56 @@
+{
+  "name": "frontend",
+  "version": "0.1.0",
+  "type": "module",
+  "private": true,
+  "scripts": {
+    "build": "next build",
+    "dev": "next dev",
+    "lint": "next lint",
+    "start": "next start",
+    "generate": "drizzle-kit generate",
+    "migrate": "drizzle-kit migrate"
+  },
+  "dependencies": {
+    "@radix-ui/react-dropdown-menu": "^2.1.1",
+    "@radix-ui/react-navigation-menu": "^1.2.0",
+    "@radix-ui/react-slot": "^1.1.0",
+    "@radix-ui/react-toast": "^1.2.1",
+    "@t3-oss/env-nextjs": "^0.10.1",
+    "add": "^2.0.6",
+    "class-variance-authority": "^0.7.0",
+    "clsx": "^2.1.1",
+    "drizzle": "^1.4.0",
+    "drizzle-orm": "^0.31.4",
+    "geist": "^1.3.1",
+    "lucide-react": "^0.381.0",
+    "next": "^14.2.5",
+    "postgres": "^3.4.4",
```

Comment:
Correct me if I'm wrong, but I don't think we should need postgres installed anymore. This may get automatically deleted once you remove any unnecessary files that are still importing it.